test/e2e/resources/mockFunctions.brs

```diff
@@ -0,0 +1,9 @@
+sub Main()
```

Comment:
I think it would be valuable to add a unit test in the components too. For example, defining a function in a component and confirming that the mock replaces the functionality

Reply:
I agree, if only because of how tricky it was to get the scoping right for scenegraph nodes 😃   This should work as-is, but it'd be nice to check!

Reply:
we might have to sync up on this because I can't quite picture how this test would be set up :/